android/src/main/kotlin/com/fingerprintjs/flutter/fpjs_pro/fpjs_pro_plugin/FpjsProPlugin.kt

```diff
@@ -110,31 +113,53 @@ class FpjsProPlugin: FlutterPlugin, MethodCallHandler {
   }
 
   private fun getVisitorId(
+    timeoutMillis: Int?,
     linkedId: String,
     tags: Map<String, Any>,
     listener: (String) -> Unit,
     errorListener: (String, String) -> (Unit)
   ) {
-    fpjsClient.getVisitorId(
-      tags,
-      linkedId,
-      listener = {result -> listener(result.visitorId)},
-      errorListener = { error -> errorListener(getErrorCode(error), error.description.toString())}
-    )
+    if (timeoutMillis != null) {
+      fpjsClient.getVisitorId(
+        timeoutMillis,
+        tags,
+        linkedId,
+        listener = { result -> listener(result.visitorId) },
+        errorListener = { error -> errorListener(getErrorCode(error), error.description.toString()) }
+      )
+    } else {
+      fpjsClient.getVisitorId(
+        tags,
+        linkedId,
+        listener = { result -> listener(result.visitorId) },
+        errorListener = { error -> errorListener(getErrorCode(error), error.description.toString()) }
+      )
+    }
```

Comment:
Wouldn't it be possible to pass `timeoutMillis` as null instead of having if/else branch where we either pass it or no?

Applies to method below as well